File-Formats/DocIO/Working-with-Find-and-Replace.md

```diff
@@ -799,6 +799,117 @@ Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Replace.docx", "applic
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Find-and-Replace/Find-and-replace-paragraphs-with-text).
 
+## Find next and replace with formatted text
+You can find the next occurrence of a text using the `FindNext`method. You can also replace the text that extends to two paragraphs using `ReplaceSingleline`method.
```

Comment:
@sharonaumao - It seems you didn't added the proper spacing between method name and sentence. Please change it properly.

Reply:
@Ramaraj-Marimuthu -I have modified